src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -151,7 +151,7 @@ def self.new_from_storage(device)
       #   <drive> section, like a disk, a DASD or an LVM volume group.
       # @return [Boolean] if attributes were successfully read; false otherwise.
       def init_from_device(device)
-        if device.is?(:md)
+        if device.is?(:software_raid)
           init_from_md(device)
```

Comment:
NP: Is the `:md` symbol still valid actually? I mean was that a typo or the check was wrong?

What about renaming `init_from_md` to `init_from_software_raid` to be consistent? Or is the `init_from_md` name OK?

Reply:
Yes, it's valid to check for `:md`. It will tell you if a device is an MD RAID. But many MD RAIDs are not defined by software. If an MD RAID is defined by BIOS, then we should export it as a regular disk, not in the format used in AutoYaST to define MDs.

Another alternative would have been:
```ruby
if device.is?(:md) && device.software_defined?
```

But I considered the other to be more concise.

Reply:
And about `init_from_md`... I wanted to keep the changes to a minimum. After all, it will still generate an entry of type CT_MD in the profile. Unfortunately, the AutoYaST profile are far from being precise when using storage-related terms. In AutoYaST, MD RAID == software RAID. AutoYaST makes no difference. Even if in reality, the set of software RAIDS are a subset of all the MD RAIDs.